types/bpmn-moddle/index.d.ts

```diff
@@ -1,31 +1,23 @@
-// Type definitions for bpmn-moddle 5.1
+// Type definitions for bpmn-moddle 8.0
 // Project: https://github.com/bpmn-io/bpmn-moddle
 // Definitions by: DefinitelyTyped <https://github.com/DefinitelyTyped>
+//                 Artur Avetisov <https://github.com/Droid0012>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 5.2
```

Comment:
What's the reason to using latest TS? (can this be done in older syntax?)

Reply:
I'm not sure I see what syntax warrants changing the minimum supported TypeScript version, it doesn't seem necessary.

Reply:
Yeah, definitely remove this line entirely.